cvmfs/loader.h

```diff
@@ -223,6 +223,7 @@ struct CvmfsExports {
     fnSaveState = NULL;
     fnRestoreState = NULL;
     fnFreeSavedState = NULL;
+    fnNeedsReadEnviron = NULL;
```

Comment:
I think we should change the version number to make sure old loaders work with newer fuse modules (and test that)

Reply:
> I think we should change the version number to make sure old loaders work with newer fuse modules (and test that)

I tried to do that, please check to see if it looks like I did it correctly.  I think I probably did something wrong, because I can smoothly upgrade from 2.11.5 to 2.12.3, but if I upgrade from 2.12.3 to my test version the reload hangs and I have to abort the fuse connection.

Reply:
I fixed the reload problem.  The fix was to move the new field to the end of the struct so it doesn't move the `cvmfs_operations` substructure.

It turns out that the new field is only used during initial load, so it is always present when it is accessed and there's no need to check for a version number change.

Downgrading from the new loader to an older cvmfs does cause a failure to unmount (and a failure to get a stack trace when the condition calls for it) if the fuse process dies with any kind of signal.  I'm not sure if that's anything to worry about though since it should be a very rare condition.